ui/src/sampleApp/README.md

```diff
@@ -0,0 +1,9 @@
+# Tanagra Sample App
```

Comment:
Not for this PR, but I think it would be helpful to centralize all our docs (e.g. either move this readme and the top-level ui one to the `docs` directory or add links to these pages in the main readme if you prefer not to move them).